src/main/drivers/gyro_sync.c

```diff
@@ -26,17 +26,9 @@
 
 volatile bool gyroDataReady = false;
 
-uint32_t gyroIntSignalledAt = 0;
-int32_t gyroIntSignalDelta = 0;
-
 void gyroSyncIntHandler(void)
```

Comment:
Now that all this `gyroSyncIntHandler` does is set a flag, it should be removed entirely. Just set the flag directyl in `mpuIntExtiHandler` instead.

So the whole `gyro_sync.c` module disappears.